autonomy_msgs/Detector[] detections
string detector_name
